sdk/common/src/main/java/io/opentelemetry/sdk/common/Clock.java

```diff
@@ -10,6 +10,15 @@
 /** Interface for getting the current time. */
 @ThreadSafe
 public interface Clock {
+
+  /**
+   * Returns a default {@link Clock} which reads from {@linkplain System#currentTimeMillis() system
+   * time}.
+   */
+  static Clock getDefault() {
```

Comment:
any thoughts to exposing a `getDefaultMonotonicClock()` method as well?

Reply:
In fact, I wonder if it might be better to *only* expose the monotonic clock, since the `now()` method on the SystemClock isn't appropriate for doing timings and will certainly be misused for that reason.

Reply:
Thinking more on this, this is also probably not a good thing to expose (read the javadoc on the MonotonicClock), so I'm not sure what the best way to expose the safest things might be. Lots of docs, I guess?

Reply:
Yeah the fact that `now()` is "duration safe in `MonotonicClock` is an implementation detail of `MonotonicClock` but not a feature of the `Clock` interface itself. I added javadoc to `now()` to clarify the pitfall, which would have applied before or after this PR.

Reply:
+1 to exposing monotonic clock...

Reply:
`MonotonicClock` is only used by the SDK (e.g., we have no user-facing builders that accept it as a parameter) so I don't think we need to add a factory method here for it. @jsuereth `MonotonicClock` is a public internal class so if you're asking for it for metrics, does it work for you already?

Reply:
Ah but that's my #3372. If you already need it then I could stop moving to package-private, that's for the other PR I guess.